_docs/sql-reference/sql-commands/007-analyze-table-refresh-metadata.md

```diff
@@ -1,35 +1,18 @@
 ---
 title: "ANALYZE TABLE REFRESH METADATA"
 parent: "SQL Commands"
-date: 2020-01-30
+date: 2020-02-17
 ---
 
 Starting from Drill 1.17, you can store table metadata (including schema and computed statistics) into Drill Metastore.
 This metadata will be used when querying a table for more optimal plan creation.
 
-The Metastore is an Alpha feature; it is subject to change. We encourage you to try it and provide feedback.
-Because the Metastore is in Beta, the SQL commands and Metastore formats may change in the next release.
-{% include startnote.html %}In Drill 1.17, this feature is supported for Parquet tables only and is disabled by default.{% include endnote.html %}
-
 To use the Drill Metastore, you must enable it at the session or system level with one of the following commands:
 
 	SET `metastore.enabled` = true;
 	ALTER SYSTEM SET `metastore.enabled` = true;
 
-Alternatively, you can enable the option in the Drill Web UI at `http://<drill-hostname-or-ip-address>:8047/options`.
-
-Once you enable the Metastore, the next step is to populate it with data. Drill can query a table whether that table
- has a Metastore entry or not. (If you are familiar with Hive, then you know that Hive requires that all tables have
- Hive Metastore entries before you can query them.) In Drill, only add data to the Metastore when doing so improves
- query performance. In general, large tables benefit from statistics more than small tables do.
-
-Unlike Hive, Drill does not require you to declare a schema. Instead, Drill infers the schema by scanning your table 
- and computes some metadata like MIN / MAX column values and NULLS COUNT designated as "metadata" to be able to
- produce more optimizations like filter push-down, etc. If `planner.statistics.use` option is enabled, this command
- will also calculate and store table statistics into Drill Metastore.
-
-Unlike Hive, Drill does not require you to declare a schema. Instead, Drill infers the schema by scanning your table
- in the same way as it is done during regular select.
+Please refer to [Using Drill Metastore]({{site.baseurl}}/docs/using-drill-metastore) for more details about Drill Metastore, its purpose and usage information. 
```

Comment:
Perhaps: "...about Drill Metastore including its purpose and how to use it."

Reply:
Thanks, reworded.